### PR TITLE
Giant lollipops!!!

### PR DESCRIPTION
**Pull request by ATDoop**
giant cartoon lollipops yippee!!!!!!! 
they can be made with a skewer and 10 units of sugar cooked for 5 seconds in a microwave!
image

:cl:
- add: Added giant lollipops!!! They can be made by cooking a skewer and 10 units of sugar for 5 seconds in a microwave.

### CODE REVIEW

File: Resources/Prototypes/Entities/Objects/Consumable/Food/meals.yml [hunk 1]
```diff
@@ -673,3 +673,30 @@
           Quantity: 4
         - ReagentId: Vitamin
           Quantity: 4
+
+- type: entity
```
Comment (hivehum) [inline Resources/Prototypes/Entities/Objects/Consumable/Food/meals.yml:677]: this should be in our file structure. so you want to make Resources/Prototypes/_Impstation/Entities/Objects/Consumable/Food/meals.yml and paste this entity there

File: Resources/Prototypes/Flavors/flavors.yml [hunk 2]
```diff
@@ -194,6 +194,11 @@
   flavorType: Base
   description: flavor-base-chewy
 
+- type: flavor
```
Comment (hivehum) [inline Resources/Prototypes/Flavors/flavors.yml:197]: same deal regarding the file structure in this and every other file. if you have questions shoot them @ me or post them in dev chat

**Comment by hivehum** [comment]
wtf is that yaml linter error

**Comment by TGRCdev** [comment]
NuGet moment?

**Review by hivehum — APPROVED** [review]
**YAY!!!**

**Pull request merged**